_format/1.1/index.md

```diff
@@ -613,6 +634,12 @@ implements at least version 1.0 of the specification.
 > Note: Because JSON:API is committed to making additive changes only, the
 version string primarily indicates which new features a server may support.
 
+### <a href="#document-fragment-syntax" id="document-fragment-syntax" class="headerlink"></a> URI Fragment Syntax
+
+A JSON:API URI fragment **MUST** be a JSON Pointer [[RFC6901](https://tools.ietf.org/html/rfc6901)] which points to a
+[resource object](resource objects) or [relationships object](relationships) in the response document and it **MUST NOT**
```

Comment:
Is there a use case to point to a resource identifier inside of the relationship object?

Reply:
I don't believe so since resource identifier objects can't contain links.